include/deal.II/base/vectorization.h

```diff
@@ -311,7 +347,7 @@ class VectorizedArray
  * @relates VectorizedArray
  */
 template <typename Number>
-inline
+inline DEAL_II_ALWAYS_INLINE
```

Comment:
I think that's fine. In `include/deal.II/base/tensor_accessors.h` or `include/deal.II/base/tensor.h` where this is also used, we always use the idiom `inline DEAL_II_ALWAYS_INLINE`.